File: src/big-picture-ui/src/model/oracle/node/column.ts
import Node from '../../base/node';

export class Column extends Node {
  DatabaseName: string;
  Schema: string;
  TableName: string;

  ColumnId: number;
  DataType: string;
  DataLength: number;
  DataPrecision: number;
  DataScale: number;
  Nullable: string;
}

File: src/BigPicture/BigPicture.Resolver.Oracle/Nodes/Column.cs
using BigPicture.Core;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BigPicture.Resolver.Oracle.Nodes
{
    public class Column : Node
    {

        public String DatabaseName { get; set; }
        public String Schema { get; set; }
        public String TableName { get; set; }

        public Int32 ColumnId { get; set; }
        public String DataType { get; set; }
        public Int32 DataLength { get; set; }
        public Int32 DataPrecision { get; set; }
        public Int32 DataScale { get; set; }
        public String Nullable { get; set; }
    }
}
